# 사람 탐지 정보
std_msgs/Header header
string          person_id
uint32          x_min
uint32          y_min
uint32          x_max
uint32          y_max
geometry_msgs/Point centroid
float32         confidence
string          robot_id
